pkg/liqo-controller-manager/foreign-cluster-operator/foreign-cluster-controller.go

```diff
@@ -318,8 +318,13 @@ func (r *ForeignClusterReconciler) peerNamespaced(ctx context.Context,
 		return err
 	}
 
+	resourceOffer, err := r.getResourceOffer(ctx, foreignCluster)
```

Comment:
this function gets the incoming resourceOffer, here we need the outgoing

this cannot work in the correct way, it is linking to the wrong resource during a bidirectional peering and it does not work at all during a unidirectional peering